policy-test/src/curl.rs

```diff
@@ -241,4 +249,36 @@ impl Running {
 
         code
     }
+
+    async fn inits_complete(&self) {
+        let api = kube::Api::namespaced(self.client.clone(), &self.namespace);
+        let init_complete = kube::runtime::wait::await_condition(
+            api.clone(),
```

Comment:
Do we need a clone here? We don't seem to make use of `api` anywhere else in the function and it'll be dropped when the function returns, would it make more sense to just move it here instead of cloning?